projects/Glados-PortalNetwork.md

```diff
@@ -0,0 +1,76 @@
+
+
+- Tests: When we encounter a bug or problem within glados it sometimes may stop glados from working or stop us from being able to view the health of the portal network which can sometimes be a problem in its own, so it is sometimes a good idea to write tests to ensure to do not encounter these same bugs again which can be very important.
+
+
+- UI: Create a ruff daft using figma of how I would improve the ui run it over with the Portal Network team and if they approve of it begin styling it in my ide.
+
+
+- Dynamic loading: Instead of everything loading all at once, it will load one thing at a time to improve performance and time complexity.
+
+
+- (If I have extra time) Lower Level Features: This is to be determined on how my progress goes during the fellowship. But would be a bonus depending how many project goes and would be a growth opportunity for me.
+
+## Road Map
+
+- 1st Phase: Research a ton into the ethereum, the portal network, glados and into my project. Continue to contribute to glados
+
+
+- 2nd: Phase: Have a really good idea into what I’m to do for my project and have completed my project proposal, as well as continue to contribute and improve glados in its entirety.
+
+
+- 3rd: Phase: By this point I should have made 3 to 4 significant contributions which has improved glados overall
+
+
+- 4th: Phase: By this point I should have made 5 to 7 significant contributions in which has improved glados overall
+
+
+- 5th: Phase: By this point I should have made 10 to 12 significant contributions which improved glados overall and created many way’s for the users to be able to easily view and visualize the health the portal network through the contributions I have done on glados. 
+
+
+- 6th: (Optional Phase if I have extra time): deep dive into the lower level workings of glados to add new features or optimizations
+
+## Possible Challenges
+
+- Some of the possible challenges I believe that I may occur are facing some learning curves along the way which may prevent me from getting PR’s out as fast and temporary halt my learning and progress along the way.
+
+
+- Working on contributions that when reviewed will be merged and are useful, while trying not to spend all my time on something that in the end might not be implemented
+
+## Goal of the project
+
+To me success is to have tried my hardest and have to have put in a ton of work and to have learned a lot along the way. My goal for this years cohort and my project is to have made many useful contributions to glados and by the end I hope I will have learned a ton and gained many skills, such as becoming really good at writing tests and becoming really good at being able to view and understand all the data. Additionally becoming really good at problem solving/research along the way.
```

Comment:
I love that your goal is to make useful contributions. The idea for this section is for you to outline what the tangible goals for the project are. If someone who knows nothing about you or the project outside of this proposal is to look at this, how will they know if you have succeeded?